tools/clang/unittests/DxrFallback/test_DxrFallback.cpp

```diff
@@ -161,8 +161,8 @@ class Tester {
   }
 
 protected:
-  DxcDllSupport m_dxcSupport;
-  DxcDllSupport m_dxrFallbackSupport;
+  LibraryDllLoader m_dxcSupport;
```

Comment:
This one is a little less obvious about it, but it's also only ever used for dxcompiler.dll